node/src/main.rs

```diff
@@ -333,6 +336,14 @@ async fn main() {
                 start_block_ingestor(&logger, block_polling_interval, ethereum_chains);
             }
 
+            if near_chains.len() > 0 {
```

Comment:
This check can be removed since `start_firehose_block_ingestor` is a noop if `near_chains` is empty. It will just log "starting 0 chains" but that's ok imo